server/routes/pages.js

```diff
@@ -47,7 +63,22 @@ async function handlePage(req, res) {
   if (page === 'categories' || page === 'index') {
     const tree = await getTree()
     const categories = buildDisplayCategories(tree)
-    res.render(template, {...categories, template: stringTemplate})
+    res.format({
+      html: () => {
+        res.render(template, {...categories, template: stringTemplate})
+      },
+
+      json: () => {
+        res.json(categories.all.map((category) => ({
+          url: category.path,
+          title: category.prettyName,
+          lastUpdatedBy: (category.lastModifyingUser || {}).displayName,
+          modifiedAt: category.modifiedTime,
+          createdAt: category.createdTime,
+          id: category.id
```

Comment:
```suggestion
          id: category.id,
          resourceType: category.resourceType
```

Here's a suggestion if you'd like to make this response a consistent shape with the search endpoint, @tilgovi, per [this thread](https://github.com/nytimes/library/pull/250/files#r578672009).